docs/update_repo.md

```diff
@@ -0,0 +1,29 @@
+# How to Update Repository
+
+This repository is a branded version of the [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) and requires some consideration when updating.
+The basic update is to merge in upstream changes from a target release version and then apply branding. However, new roles may need additional testing added to them.
+
+```
+git checkout -b update-to-<target tag>
+git remote add upstream https://github.com/theforeman/foreman-ansible-modules
+git fetch upstream
+git merge upstream/<target tag>
```

Comment:
I might be wrong (and I didn't try it right now), but I think tag objects are "global", and you can refer to them using the `remote/thing` notation. So this should be `git merge <target tag>`

Reply:
Oh yes, they are, and this notation does not work. I just copied it from the other doc and well never applied my local griping to the change :D